src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -0,0 +1,411 @@
+            </div>
+          </div>
+        </ng-container>
+
+        <!-- RGW -->
+        <ng-container *ngIf="!serviceForm.controls.unmanaged.value && serviceForm.controls.service_type.value === 'rgw'">
+          <!-- rgw_frontend_port -->
+          <div class="form-group row">
+            <label i18n
+                   class="cd-col-form-label"
+                   for="rgw_frontend_port">Port</label>
+            <div class="cd-col-form-input">
+              <input id="rgw_frontend_port"
+                     class="form-control"
+                     type="number"
+                     formControlName="rgw_frontend_port"
+                     min="1"
+                     max="65535">
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('rgw_frontend_port', frm, 'pattern')"
+                    i18n>The entered value needs to be a number.</span>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('rgw_frontend_port', frm, 'min')"
+                    i18n>The value must be at least 1.</span>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('rgw_frontend_port', frm, 'max')"
+                    i18n>The value cannot exceed 65535.</span>
+            </div>
+          </div>
+        </ng-container>
+
+        <!-- iSCSI -->
+        <ng-container *ngIf="!serviceForm.controls.unmanaged.value && serviceForm.controls.service_type.value === 'iscsi'">
+          <!-- pool -->
+          <div class="form-group row">
+            <label i18n
+                   class="cd-col-form-label required"
+                   for="pool">Pool</label>
+            <div class="cd-col-form-input">
+              <select id="pool"
+                      name="pool"
+                      class="form-control custom-select"
+                      formControlName="pool">
+                <option *ngIf="pools === null"
+                        [ngValue]="null"
+                        i18n>Loading...</option>
+                <option *ngIf="pools !== null && pools.length === 0"
+                        [ngValue]="null"
+                        i18n>-- No pools available --</option>
+                <option *ngIf="pools !== null && pools.length > 0"
+                        [ngValue]="null"
+                        i18n>-- Select a pool --</option>
+                <option *ngFor="let pool of pools"
+                        [value]="pool.pool_name">{{ pool.pool_name }}</option>
+              </select>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('pool', frm, 'required')"
+                    i18n>This field is required.</span>
+            </div>
+          </div>
+
+          <!-- trusted_ip_list -->
+          <div class="form-group row">
+            <label class="cd-col-form-label"
+                   for="trusted_ip_list">
+              <span i18n>Trusted IP's</span>
```

Comment:
```suggestion
              <span i18n>Trusted IPs</span>
```

The Trusted IPs need to be MGR's IPs. Otherwise, the iSCSI page of the Dashboard can't work because mgr can't reach iSCSI gateway daemons. Maybe we can enhance this in the help text or make the field required.

Reply:
I've added a help text, but didn't mark the field as `required` because i'm not sure if there are situations where it makes sense to do not set any IP here.